### ISSUE TITLE
Artillery playwright config does not allow 0 timeout

### ISSUE THREAD

**Issue by HugoImaios** [artilleryio/artillery#2785]
When setting a `0` value for defaultNavigationTimeout, defaultPageTimeout or defaultTimeout, artillery fallback on 30s.

Version info: 
```
v2.0.12
```

Using this config: 
```
            defaultNavigationTimeout: 0
            defaultTimeout: 0
```

I expected to see this happen:

having a 0 timeout (no timeout)

Instead, this happened:

30s timeout is used

**Comment by HugoImaios** [comment]
it comes from https://github.com/artilleryio/artillery/blob/c8adbb73a431431430a96203b8dac432526390cc/packages/artillery-engine-playwright/index.js#L16 (and https://github.com/artilleryio/artillery/blob/c8adbb73a431431430a96203b8dac432526390cc/packages/artillery-engine-playwright/index.js#L18) : `0` is falsy so `||` will return the right operand, the default `30`

**Comment by hassy** [comment]
falsy values strike again! mind sending a quick PR? :)

**Comment by HugoImaios** [comment]
Yup ! I did the fork, I'll open one on monday, I need 0 timeout lol
do you prefere a one-liner style or a cleaner const one ?
